source/settings/settings-reference.txt

```diff
@@ -86,6 +86,17 @@ You can configure the following settings on the |compass| interface:
 
        To learn more, see :ref:`compass-enable-dev-tools`.
 
+   * - Install |compass-short| as URL Protocol Handler
+     - General
+     - Register |compass-short| as a handler for mongodb:// and mongodb+srv:// 
+       URLs. 
+
+       If :guilabel:`Install Compass as URL Protocol Handler` is enabled, 
+       you can open |compass-short| by navigating to a mongodb:// or 
+       mongodb+srv:// URL.
```

Comment:
**Suggestion**

Add "in your browser" to the end of the sentence to clarify the behavior.

Reply:
Noted and updated!